.circleci/config.yml

```diff
@@ -372,6 +372,25 @@ jobs:
             docker tag $IMAGE_NAME:$DOCKER_TAG $ECR_REPO/$IMAGE_NAME:$DOCKER_TAG
             docker push $ECR_REPO/$IMAGE_NAME:$DOCKER_TAG
 
+  # new aws env
+  build_and_push:
+    environment:
+      IMAGE_NAME: tariff-frontend
+    parameters:
+      environment:
+        type: string
+    docker:
+      - image: cimg/ruby:3.2.2-node
```

Comment:
I wonder if we could move to a smaller image?

Reply:
It has to build the application - `cimg/ruby:3.2.2-node` happens to have all the binaries we need for doing that. Is there any other image we could use to build?